templates/patterns/card/pattern-card--variant-search.html.twig

```diff
@@ -5,58 +5,54 @@
  */
 #}
 {% spaceless %}
-  {% set _badges = [] %}
-  {% if badges is not empty %}
-    {% set badge_attr = create_attribute({'class': ['me-2']}) %}
-    {% for key,item in badges %}
-      {% if not loop.last %}
-        {% set badge_attr = create_attribute() %}
-      {% endif %}
-      {% if key|first != '#' %}
-        {% set _badges = _badges|merge([{
-          label: item['#title'],
-          outline: true,
-          rounded_pill: true,
-          attributes: badge_attr
-        }]) %}
-      {% endif %}
-    {% endfor %}
+{% set content %}
+  {% if meta is not empty %}
+    {# Use negative margin to compensate for me-3 on each item. #}
+    <div class="my-3 me-n3">
+      {% for _meta_item in meta %}
+        <span class="text-muted me-3">{{ _meta_item }}</span>
+      {% endfor %}
+    </div>
   {% endif %}
-  {% set _image = image %}
-  {% if image is defined and image['#markup'] is not empty %}
-    {% set _image = {
-      path: image['#markup'],
-      position: 'top',
-      classes: 'd-none d-md-block',
-    } %}
-  {% endif %}
-  {% if text is defined and text.classes is not defined %}
-    {% set text = {
-      content: text|render|striptags,
-      classes: 'mb-2-5',
-    } %}
-  {% endif %}
-  {% include '@oe-bcl/card' with {
-    horizontal: true,
-    horizontal_grid: {
-      left_col_classes: 'col-md-3 col-lg-2 rounded mw-listing-img',
-      right_col_classes: 'col-md-9 col-lg-10',
-    },
-    extra_classes_body: 'p-0 pb-md-0 pb-3',
-    badges: _badges,
-    title: {
-      content: title,
-    },
-    text: text,
-    attributes: create_attribute({'class': [
-      'listing-item',
-      'border-bottom',
-      'border-md-0',
-      'border-0',
-      'mb-3',
-    ]}),
-    text_color: 'dark',
-    image: _image,
-    content: content
-  } only %}
+  <div>{{ content }}</div>
+{% endset %}
+{% set _badges = [] %}
+{% if badges is not empty %}
+  {% for item in badges %}
+    {% set _badges = _badges|merge([{
+      outline: true,
+      rounded_pill: true,
+    }|merge(item)]) %}
+  {% endfor %}
+{% endif %}
+{% include '@oe-bcl/card' with {
+  horizontal: true,
```

Comment:
In the other template we quote the keys, in here we don't. We should stick to one, I believe we went for always quote.